docs/configuration.md

```diff
@@ -3698,6 +3698,13 @@ The section contains the following settings (available since v3.8.1):
     - other flags below might extend rewrites to other cases
 - `oldSyntaxToo`
   - if `true`, applies also to expressions using deprecated syntax
+- `fewerBracesMinNonWsSpan` and `fewerBracesMaxNonWsSpan`
```

Comment:
```suggestion
- `fewerBracesMinSpan` and `fewerBracesMaxSpan`
```
I would try to make it a bit less explicit, if anyone needs to check what is exactly being done they will anyway have to check the docs.

Reply:
thank you, done!

Reply:
Thanks!